plugins/Satellites/src/Satellites.hpp

```diff
@@ -345,11 +347,10 @@ class Satellites : public StelObjectModule
 	//! \param openFile a reference to an \b open file.
 	//! @param[in,out] tleList a hash with satellite IDs as keys.
 	//! @param[in] addFlagValue value to be set to TleData::addThis for all.
+	//! @param[in] sourceTLE a source of TLE (e.g. Celestrak's URL)
```

Comment:
again name: if this is an URL, reflect this in the var name. maybe tleUrl?

Reply:
OK